nist.c

```diff
@@ -0,0 +1,135 @@
+#include <oqs/common.h>
+#include <stdio.h>
+#include "api.h"
+#include "params.h"
+#include "nist_params.h"
+#include "xmss.h"
+
+/*************************************************
+ * Name:        crypto_sign_keypair
+ *
+ * Description: Generates public and private key.
+ *
+ * Arguments:   - uint8_t *pk: pointer to output public key (allocated
+ *                             array of LMS_CRYPTO_PUBLICKEYBYTES bytes)
+ *              - uint8_t *sk: pointer to output private key (allocated
+ *                             array of LMS_CRYPTO_SECRETKEYBYTES bytes)
+ *
+ * Returns 0 (success), -1 otherwise
```

Comment:
Are the output arrays left unchanged on error? Is it specified?

Reply:
Also I don't think it's a good idea to duplicate the documentation between the .h and the .c files.

Reply:
I clean up documentation in .c file